### PR TITLE
bugfix: fix rename's bug

### PR DESCRIPTION
**Pull request by skoo87**
the rename can't work.

Signed-off-by: skoo87 <marckywu@gmail.com>

### CODE REVIEW

**Comment by zeppp** [comment]
I think it caused by `r.Path("/containers/{name:.*}/rename").Methods(http.MethodPost).Handler(s.filter(s.renameContainer))` which in /apis/server/router.go  
Maybe we should change it into `r.Path("/containers/{id:.*}/rename").Methods(http.MethodPost).Handler(s.filter(s.renameContainer))` 
so we can keep consistent with it defination in swagger.yml
@skoo87

**Comment by janynedoe48** [comment]
Why can't I change my Name(s)?  About an hour ago I was on goo.** maps and it showed my location at some other spot?  This is noT THe first time.  THIS IS CREEPY DEFINATLEY CREEPY AS THEY ARE SHOWING FRONT DOORS OF HOMES AND WITH THE QUICKNESS. SOMEHOW ITS Blocking me because of ABUSE! Which end of the report am I??? NOT from *MY CARRIER EITHER! [JayneDoe48 IS ME ALSO] This current event occurred (when I) used a different means of accessing my location.  <NOW I AM NILL? WT*? B4 NU**!  Also FUL***IGHT SAYS MY LAST NAME IS "I"..CAN SOMEBODY TURN OFF DEV FOR ME ACROSS THE WEB!!! I M NOT A DEV NOR A NULL NOR A NOBODY THEY CALL ME SOX WHERE I STAY....BARSTOW.  (Ju*p! ** **mand/**r*a*# ***2**0856**446)=me2! Thanx for helping me..pss

**Comment by janynedoe48** [comment]
Why can't I change my Name(s)?  About an hour ago I was on goo.** maps and it showed my location at some other spot?  This is noT THe first time.  THIS IS CREEPY DEFINATLEY CREEPY AS THEY ARE SHOWING FRONT DOORS OF HOMES AND WITH THE QUICKNESS. SOMEHOW ITS Blocking me because of ABUSE! Which end of the report am I??? NOT from *MY CARRIER EITHER! [JayneDoe48 IS ME ALSO] This current event occurred (when I) used a different means of accessing my location.  <NOW I AM NILL? WT*? B4 NU**!  Also FUL***IGHT SAYS MY LAST NAME IS "I"..CAN SOMEBODY TURN OFF DEV FOR ME ACROSS THE WEB!!! I M NOT A DEV NOR A NULL NOR A NOBODY THEY CALL ME SOX WHERE I STAY....BARSTOW.  (Ju*p! ** **mand/**r*a*# ***2**0856**446)=me2! Thanx for helping me..pss

**Comment by skoo87** [comment]
I agree with you, but the variable "id" in mgr.Rename() is not a good name, I think it should be consistent with other interfaces.  @zeppp

File: daemon/mgr/container.go [hunk 1]
```diff
@@ -50,7 +50,7 @@ type ContainerMgr interface {
 	Remove(ctx context.Context, name string, option *ContainerRemoveOption) error
 
 	// Rename renames a container
-	Rename(ctx context.Context, id string, name string) error
+	Rename(ctx context.Context, oldName string, NewName string) error
```
Comment (allencloud) [inline daemon/mgr/container.go]: s/NewName/newName to keep consistent

**Comment by zeppp** [comment]
Yes, the oldname is better.I am sorry to push unchecked code.
@skoo87

**Comment by allencloud** [comment]
LGTM

**Pull request merged**